docs/source/en/peft.md

```diff
@@ -42,6 +40,26 @@ If you want to try out the brand new features, you might be interested in instal
 pip install git+https://github.com/huggingface/peft.git
 ```
 
+## Load PEFT adapter
+
+To load and use a PEFT adapter model from 🤗 Transformers, make sure the Hub repository or local directory contains an `adapter_config.json` file and the adapter weights, as shown in the example image above. Then you can load the PEFT adapter model using the `AutoModelFor` class. For example, to load a PEFT adapter model for causal language modeling:
+
+1. specify the PEFT model id
+2. pass it to the [`AutoModelForCausalLM`] class
+
+```py
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+peft_model_id = "ybelkada/opt-350m-lora"
+model = AutoModelForCausalLM.from_pretrained(peft_model_id)
+```
+
+<Tip>
+
+You can still load a PEFT adapter even if your model isn't an auto-mapped model as long as the Hub repository or your local directory contains an `adapter_config.json` file with the `base_model_name_or_path` key.
+
+</Tip>
+
 <!--
 
 TODO: (@younesbelkada @stevhliu)
```

Comment:
For the `Trainer` integration, did you mean something like:

```py
model_name = "roberta-large"
peft_config = PromptEncoderConfig(task_type="SEQ_CLS", num_virtual_tokens=20, encoder_hidden_size=128)
model = AutoModelForSequenceClassification.from_pretrained(model_name)
model = get_peft_model(model, peft_config)
trainer = Trainer(model=model, ...)
trainer.train()
```

Reply:
Exactly yes ! Although it seems most of the usage seems to be around LoRA models , but overall yes if you pass a `PeftModel` to trainer it should just work like that